src/idls/xtc.did.ts

```diff
@@ -41,8 +50,10 @@ export default ({ IDL }) => {
   });
   return IDL.Service({
     meta: IDL.Func([], [TokenMetaData], ['query']),
+    getMetadata: IDL.Func([], [Metadata], ['query']),
     meta_certified: IDL.Func([], [TokenMetaData], []),
     balance: IDL.Func([IDL.Opt(IDL.Principal)], [IDL.Nat64], []),
```

Comment:
there shouldn't be any balance now, I think the IDL should be looked over more carefully

Reply:
I cannot remember all the changes from the top of my head